src/equations/numerical_fluxes.jl

```diff
@@ -243,7 +245,7 @@ Base.show(io::IO, numflux::FluxHLL) = print(io, "FluxHLL(", numflux.min_max_spee
 See [`FluxHLL`](@ref).
 """
 const flux_hll = FluxHLL()
-
+const flux_hll_cn = FluxHLLChenNoelle()
```

Comment:
```suggestion
const flux_hll_chen_noelle = FluxHLLChenNoelle()
```
I would give this new HLL-type flux a more descriptive name.